src/live_view.cpp

```diff
@@ -36,11 +36,21 @@ int live_view::draw( const catacurses::window &win, const int max_height )
 
     // -1 for border. -1 because getmaxy() actually returns height, not y position.
     const int line_limit = max_height - 2;
```

Comment:
```suggestion
     const size_t line_limit = max_height - 2;
```

Please fix data type of `line_limit` to unsigned (e.g. `size_t`) to fix warning in comparison

```
src/live_view.cpp:46:21: error: comparison of integer expressions of different signedness: 'std::vector<std::__cxx11::basic_string<char> >::size_type' {aka 'long long unsigned int'} and 'const int' [-Werror=sign-compare]
     if( text.size() > line_limit ) {
         ~~~~~~~~~~~~^~~~~~~~~~~~
cc1plus.exe: all warnings being treated as errors
make: *** [Makefile:813: objwin/tiles/live_view.o] Error 1
```